packages/material-ui/src/test-utils/findOutermostIntrinsic.test.js

```diff
@@ -20,6 +21,18 @@ describe('findOutermostIntrinsic', () => {
   };
   const Headless = ({ children }) => children;
 
+  before(() => {
+    // Trigger deprecation warning.
+    try {
+      stub(console, 'warn');
+      findOutermostIntrinsic();
+    } catch (error) {
+      // ignore
+    } finally {
+      console.warn.restore();
+    }
+  });
+
```

Comment:
Ultimately we'd like to reset modules between each test so that we could test this properly. But we don't have any such API (like `jest.resetModules`). Though we might want to add per-module methods that manually take care of this and which we strip from the published code. Just  typing this out loud for future reference. Right now I don't think it's worth it.